### PR TITLE
Add HTML structure and required functionalities

### PR DESCRIPTION
**Pull request by enionsouza**
In this milestone, the following files were created in the `/src` folder:
- [x] `index.html`;
- [x] `index.js`;
- [x] `task.js`; and
- [x] `style.css`.

Additionally, an array of tasks was created and successfully stored in the `localStorage` object.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,44 @@
...
+  }
+}
 
-const logo = document.createElement('img');
-logo.src = WebpackLogo;
+const myToDoList = new ToDo();
+if (localStorage.tasks === undefined) {
+  myToDoList.addTask('My first task');
+  myToDoList.addTask('My second task');
+  myToDoList.addTask('My third task');
```
Comment (bellom) [inline src/index.js:41]: Let's create an array and iterate over the array as stated in the project instructions.
Screenshot 👇 
image

File: src/index.js [hunk 2]
```diff
@@ -1,10 +1,44 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-console */
+
 import './style.css';
```
Comment (bellom) [inline src/index.js:6]: Let's create the minimal styling, our structure should be like this 👇 
image


In another word; let's try to clone the example given on the project page above.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @enionsouza  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

**_Kindly check the comments under the review._**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by enionsouza** [comment]
### 2nd Code Review
Hopefully, I was able to perform all the previously required changes :grimacing: 
- [x] The functionality for creating and iterating over an array of objects was done; and
- [x] A minimal structured layout was created by providing styles to the app.

**Review by ZawadiSumba66 — APPROVED** [review]
Hi @enionsouza 👋 , 
## STATUS: APPROVED 👍 
Great job implementing all the changes requested by the previous reviewer. 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**